aries-site/src/examples/components/checkbox/CheckBoxDescription.js

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react';
+import { Box, CheckBox, FormField } from 'grommet';
+
+export const CheckBoxDescriptionExample = () => {
```

Comment:
The convention we have been using is to have the filename consistent with the component it is exporting.